spec/views/alchemy/ingredients/link_editor_spec.rb

```diff
@@ -26,12 +26,11 @@
 
   it_behaves_like "an alchemy ingredient editor"
 
-  it "renders a disabled text input field" do
-    is_expected.to have_selector('input[type="text"][disabled]')
+  it "renders a readonly text input field" do
+    is_expected.to have_selector('input[type="text"][readonly]')
```

Comment:
You're probably aware and this might be fully intentional, but let me ask: changing from disabled to readonly makes it behave differently as disabled inputs aren't sent while readonly inputs are. Any impact due to this? If this is intentional and okay, maybe it's worth mentioning?

Reply:
Yes, it is fully intentional. Before we had a disabled form field that does not get submitted with the form and a hidden field that gets submitted with the form. The disabled field was used as display that needed to get updated via JS and stopped working with the change to ingredients. Now, instead having both fields we just have one that gets submitted, but still does not allow edits by the user. The readonly field now correctly gets updated via js. So a win-win. Less code better UX :)

Reply:
Thats great, thanks for clearifying!